client/views/index.html

```diff
@@ -1,11 +1,18 @@
-<h1>Index</h1>
+<h2>Search for any food item:</h2>
+
+<fd-search get-suggestions="vm.get-suggestions()" submit="vm.submit" query="vm.form.query" suggestions="vm.search.suggestions"></fd-search>
```

Comment:
`get-suggesions` won't be a valid js property name, so you'll want to call the method getSuggestions.

Apart from that you'll want to pass the method as a callback rather than calling it. So `"vm.getSuggestions"` rather than `"vm.get-sugggestions()"`